components/suggest/src/yelp.rs

```diff
@@ -375,10 +416,25 @@ impl<'a> From<SuggestionBuilder<'a>> for Suggestion {
         }
         url.push_str(&parameters.finish());
 
+        let title = [
```

Comment:
Same here:

```rs
        let title = [
            builder.pre_yelp_modifier.as_deref(),
            builder.pre_modifier.as_deref(),
            Some(builder.subject),
            builder.post_modifier.as_deref(),
            builder.location_sign.as_deref(),
            builder.location.as_deref(),
            builder.post_yelp_modifier.as_deref(),
        ]
        .iter()
        .flatten()
        .map(|&s| s)
        .collect::<Vec<_>>()
        .join(" ");
```

Reply:
A lint error happens from `rust-clippy`.
https://app.circleci.com/pipelines/github/mozilla/application-services/36787/workflows/52aeceb3-9d98-4fe4-8037-ffccae4cd57e/jobs/186366?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary
So, as it said, I use `.flatten().copied()`.

Reply:
Ah, of course, thanks Clippy!